managed/services/backup/backup_service.go

```diff
@@ -356,6 +373,11 @@ func (s *Service) RestoreBackup(ctx context.Context, serviceID, artifactID strin
 			DBConfig:      dbConfig,
 			DataModel:     artifact.DataModel,
 			PITRTimestamp: pitrTimestamp,
+			Folder:        artifactFolder,
+		}
+
+		if len(artifact.StorageRecList) != 0 && artifact.StorageRecList[0].BackupRec != nil {
+			params.ArtifactSysName = artifact.StorageRecList[0].BackupRec.Name
```

Comment:
What if `StorageRecList` has more than one items?

Reply:
Than mean it is PITR artifact and name will not be used at all.